docs/detections/prebuilt-rules/downloadable-packages/0-16-2/prebuilt-rule-0-16-2-potential-lsass-memory-dump-via-psscapturesnapshot.asciidoc

```diff
@@ -0,0 +1,81 @@
+[[prebuilt-rule-0-16-2-potential-lsass-memory-dump-via-psscapturesnapshot]]
+=== Potential LSASS Memory Dump via PssCaptureSnapShot
+
+Identifies suspicious access to an LSASS handle via PssCaptureSnapShot where two successive process accesses are performed by the same process and target two different instances of LSASS. This may indicate an attempt to evade detection and dump LSASS memory for credential access.
```

Comment:
```suggestion
Identifies suspicious access to a Local Security Authority Server Service (LSASS) handle via PssCaptureSnapShot where two successive process accesses are performed by the same process and target two different instances of LSASS. This may indicate an attempt to evade detection and dump LSASS memory for credential access.
```